collector/project.go

```diff
@@ -49,17 +53,26 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 	var poUtils []float64
 	var hpaUtils []float64
 
+	// Setup vars for catalogs
+	perClusterCatalogMap := make(map[string]bool)
+	rancherCatalog, err := c.Client.Catalog.ByID("library")
+	if err != nil || rancherCatalog.URL != rancherCatalogURL {
+		log.Error("Failed to find a valid rancher default catalog")
+		rancherCatalog = nil
+	}
+
 	for _, project := range list.Data {
-		// Namespace
 		parts := strings.SplitN(project.ID, ":", 2)
+		clusterID := parts[0]
```

Comment:
Why is this section getting moved?

Reply:
To use clusterID, but realized that wouldn't work as-written. Fixed now.